src/i18n.js

```diff
@@ -1,19 +1,23 @@
 import i18n from 'i18next';
 import { initReactI18next } from 'react-i18next';
 import LanguageDetector from 'i18next-browser-languagedetector';
-import TranslationsEn from './translations/english.json';
-import TranslationsSp from './translations/spanish.json';
-
-const resources = {
-  en: { translation: TranslationsEn },
-  es: { translation: TranslationsSp },
-};
+import resourcesToBackend from 'i18next-resources-to-backend';
 
 i18n
   .use(initReactI18next)
   .use(LanguageDetector)
+  .use(
+    resourcesToBackend((lng, ns, clb) => {
```

Comment:
I suggest using slightly more readable names:
```suggestion
    resourcesToBackend((lang, ns, callback) => {
```